cmd/main.go

```diff
@@ -27,13 +28,26 @@ func init() {
 	flag.BoolVar(&config.NoTeamsAPI, "noteamsapi", false, "Disable all access to the teams API")
 	flag.Parse()
 
+	cmd := exec.Command("cat", "/var/run/secrets/kubernetes.io/serviceaccount/namespace")
```

Comment:
Don't you want [ioutil.ReadFile](https://golang.org/pkg/io/ioutil/#ReadFile) here?

Reply:
I'd actually assume there is a MY_POD_NAMESPACE environment variable set to metadata.namespace and read that variable.

Reply:
>  I'd actually assume there is a MY_POD_NAMESPACE

I observe this env var neither in `minikube` nor in actual k8s deployments.

Reply:
Sure, it needs to be added to our Kubernetes deployment manifest. That's how Kubernetes Downward API work https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/

Reply:
Isnt there an env var set by default for this?

Reply:
the answer is no, as far as i can see from manual examination of the env

Reply:
```
POD_NAMESPACE=default
```

Use this if it is supplied by Kubernetes 1.8+